onnx/defs/tensor/defs.cc

```diff
@@ -735,7 +735,9 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
-static const char* Scatter_ver9_doc = R"DOC(
+static const char* Scatter_ver11_doc = R"DOC(
+This operator is deprecated. Please use ScatterElements, which provides the same functionality.
+
```

Comment:
I think it would help to share the updated documentation used for ScatterElements for Scatter also. It will help make it clear they have the same definition.

Reply:
you mean replace scatter's definition by the new definition, of just mention that the definition is updated in ScatterElements?

Reply:
I mean the first (replace the old documentation with the new documentation), with the deprecation line alone being extra.